geode-docs/managing/security/implementing_authentication.html.md.erb

```diff
@@ -23,151 +23,150 @@ Authentication lends a measure of security to a cluster
 by verifying the identity of components as they connect to the system.
 All components use the same authentication mechanism.
 
-## How Authentication Works
+## <a id="authentication-how-it-works"></a>How Authentication Works
 
 When a component initiates a connection to the cluster,
 the `SecurityManager.authenticate` method is invoked.
 The component provides its credentials in the form of properties
 as a parameter to the `authenticate` method.
-The credential is presumed to be the two properties
+The credentials parameter is presumed to be a user/password combination, specified as the two properties
```

Comment:
they don't have to be user/password combinations, they could be tokens, or certificates or username/password combinations.

Reply:
Thanks, @jinmeiliao Change incorporated.